command/ca/token.go

```diff
@@ -388,6 +374,67 @@ func newTokenFlow(ctx *cli.Context, subject string, sans []string, caURL, root,
 	return generateToken(subject, sans, kid, issuer, audience, root, notBefore, notAfter, jwk)
 }
 
```

Comment:
```
// offlineTokenFlow generates a provisioning token using  either 
//   1. static configuration from ca.json (created with `step ca init`)
//   2. input from command line flags
// These two options are mutually exclusive and priority is given to ca.json.
```

Leave this comment above the method. Was confused since I forgot `step ca token` had an older, more complex implementation.